_data/saml.yml

```diff
@@ -0,0 +1,3 @@
+year:
+  current: 2024
+  previous: 2023
```

Comment:
love this! would it be worth putting the SAML certs in here too so there's only one file to touch every year?

Reply:
oh yeah i think that will make it super easy

Reply:
big fan big fan